### PR TITLE
Add markenzoo/contao-expo-push-notification-bundle

### PR DESCRIPTION
**Pull request by felix-kaestner**

### CODE REVIEW

**Comment by felix-kaestner** [comment]
I'm thinking maybe _React_ should instead be on the default whitelist?

File: meta/markenzoo/contao-expo-push-notification-bundle/logo.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    <path d="M331.07,122.879C331.07,112.678 322.789,104.396 312.588,104.396L94.105,104.396C83.904,104.396 75.622,112.678 75.622,122.879L75.622,270.443C75.622,280.644 83.904,288.926 94.105,288.926L182.235,288.926L208.12,320.128L234.313,288.926L312.588,288.926C322.789,288.926 331.07,280.644 331.07,270.443L331.07,122.879ZM203.231,169.279C215.813,151.516 236.529,153.16 247.628,162.326C258.728,171.492 260.653,191.137 247.628,208.155C237.653,221.189 219.88,235.653 203.231,244.819C186.583,235.653 168.627,221.904 158.835,208.155C146.403,190.7 147.735,171.492 158.835,162.326C169.934,153.16 189.274,150.27 203.231,169.279Z" style="fill:white;"/>
+</svg>
```
Comment (fritzmg) [inline meta/markenzoo/contao-expo-push-notification-bundle/logo.svg:12]: Please minify the SVG as explained in the README.

**Comment by felix-kaestner** [comment]
Added _React_ to default whitelist instead even though it does not fail for englisch since it is an englisch term.

File: meta/markenzoo/contao-expo-push-notification-bundle/logo.svg (hunk 1 above)
Comment (felix-kaestner) [reply meta/markenzoo/contao-expo-push-notification-bundle/logo.svg]: Done in ae124f3

File: meta/markenzoo/contao-expo-push-notification-bundle/en.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+en:
+    title: Contao Expo Push Notification Bundle
+    description: This bundle gives you the possibility to send custom push notifications for Expo (React Native) in Contao 4
```
Comment (fritzmg) [inline meta/markenzoo/contao-expo-push-notification-bundle/en.yml:3]:
```suggestion
    description: This extension gives you the possibility to send custom push notifications for Expo (React Native) in Contao 4.
```
Minor nitpick :)

Comment (felix-kaestner) [reply meta/markenzoo/contao-expo-push-notification-bundle/en.yml]: Yes that is better, thank you. :)

**Pull request merged**